### ISSUE TITLE
Still possible to screenshot the current buffer and hilight selection?

### ISSUE THREAD

**Issue by watzon** [krivahtoo/silicon.nvim#30]
Lost my old config somehow, but a few weeks ago I had an nvim config that had keybindings for screenshotting the complete buffer, screenshotting the selection, and screenshotting the entire buffer and highlighting the current selection. Now I can't find the latter in the readme though. Is this still an option?

**Comment by krivahtoo** [comment]
No, there was no such option. But  I think it would be nice to add to `silicon.nvim`.

**Comment by watzon** [comment]
Hmm maybe I'm just crazy then 😂 but it would definitely be a nice thing to have
